tests/extmod/vfs_fat_ramdisk.py

```diff
@@ -64,15 +64,14 @@ def ioctl(self, op, arg):
 
 try:
     vfs.rmdir("foo_file.txt")
-except OSerror as e:
-    assert e.args[0] == uerrno.ENOTDIR
+except OSError as e:
+    assert e.args[0] == 20 # ENOTDIR
```

Comment:
Not to write a test, sorry ;-). We don't want testsuite to bloat production builds ;-). (So, it can be added - when many (at least few) people report good usecases for having it.)